Resources/Computing/ComputingElement.py

```diff
@@ -206,6 +208,12 @@ def setParameters( self, ceOptions ):
         generalCEDict.update( self.ceParameters )
         self.ceParameters = generalCEDict
 
+    # If NumberOfProcessors is present in the description but is equal to zero
+    # interpret it as needing local evaluation
+    if "NumberOfProcessors" in self.ceParameters:
+      if self.ceParameters['NumberOfProcessors'] == 0:
```

Comment:
I think the `self.ceParameters['NumberOfProcessors']` should be cast to `int` or this might be always false because ceParameters just contains strings if I remember correctly.